nixos/tests/switch-test.nix

```diff
@@ -0,0 +1,25 @@
+# Mutable users tests.
+
+import ./make-test.nix ({ pkgs, ...} : {
+  name = "mutable-users";
```

Comment:
Probably not the name you intended?

Reply:
Good catch.

A more important question: do you think a separate test like this is useful?

Reply:
I'm not sure to be honest.